src/components/Button.tsx

```diff
@@ -32,15 +33,15 @@ export const Button: FunctionComponent<ButtonProps> = ({
   invert,
   testID,
 }) => {
-  const determineButtonStyle = (): ViewStyle => {
+  const determineGradient = (): [string, string] => {
     if (invert && (disabled || loading)) {
-      return style.buttonInvertedDisabled
+      return [Colors.mediumGray, Colors.lighterGray]
```

Comment:
Would it make sense to extract these tuples as constants? Probably an overkill but there's at least a couple repeated that might be tricky to realize if more changes are made in the future

Reply:
Good call! Will handle in my next PR.